src/backend/distributed/sql/udfs/repl_origin_helper/latest.sql

```diff
@@ -0,0 +1,22 @@
+
+CREATE OR REPLACE FUNCTION pg_catalog.replication_origin_session_start_no_publish()
+RETURNS void
+LANGUAGE C STRICT
+AS 'MODULE_PATHNAME', $$replication_origin_session_start_no_publish$$;
+COMMENT ON FUNCTION pg_catalog.replication_origin_session_start_no_publish()
+    IS 'To start Replication origin session for skipping publishing WAL records';
+
+CREATE OR REPLACE FUNCTION pg_catalog.replication_origin_session_end_no_publish()
+RETURNS void
+LANGUAGE C STRICT
+AS 'MODULE_PATHNAME', $$replication_origin_session_end_no_publish$$;
+COMMENT ON FUNCTION pg_catalog.replication_origin_session_end_no_publish()
+    IS 'To finish Replication origin session for skipping publishing WAL records';
+
+CREATE OR REPLACE FUNCTION pg_catalog.replication_origin_session_is_no_publish()
```

Comment:
maybe prefix with worker_ or citus_internal_ to signal that they are internal functions not meant to be called directly by users.

Reply:
Ok changed this to citus_internal_is_replication_origin_tracking_active